package.json

```diff
@@ -89,6 +89,7 @@
     "@wdio/selenium-standalone-service": "^5.16.10",
     "@wdio/utils": "^5.16.15",
     "apollo-server-express": "^2.9.16",
+    "arrify": "^2.0.1",
```

Comment:
No more package errors mentioned?

Reply:
I tried to `npm i arrify` then it worked.

Reply:
my bad, the required lib should stay in `dependencies` instead of `devDependencies`